app/src/main/java/com/celzero/bravedns/ui/HomeScreenSettingBottomSheet.kt

```diff
@@ -182,10 +188,10 @@ class HomeScreenSettingBottomSheet() : BottomSheetDialogFragment() {
 
     private fun modifyBraveMode() {
         HomeScreenActivity.GlobalVariable.appMode?.setFirewallMode(firewallMode)
-        if (HomeScreenActivity.GlobalVariable.braveMode == HomeScreenFragment.FIREWALL_MODE) {
+        if (HomeScreenActivity.GlobalVariable.braveMode == APP_MODE_FIREWALL) {
             HomeScreenActivity.GlobalVariable.appMode?.setDNSMode(Settings.DNSModeNone)
         } else {
-            HomeScreenActivity.GlobalVariable.appMode?.setDNSMode(-1)
+            HomeScreenActivity.GlobalVariable.appMode?.setDNSMode(PREF_DNS_INVALID)
```

Comment:
so, what does it mean to set `dns_invalid` mode? How does the app treat it?

Reply:
If the DNS is invalid, then the appMode will fetch mode based on DNS type.

Reply:
If the DNS is invalid, then the appMode will fetch mode based on DNS type.